strictdoc/export/html/generators/view_objects/document_screen_view_object.py

```diff
@@ -358,15 +358,15 @@ def get_stable_link(
         """
         Get a stable link for a given node.
 
-        An example of a link produced: ../../#SDOC_UG_CONTACT
+        An example of a link provided: ../../#SDOC_UG_CONTACT
```

Comment:
Now that I am re-reading this, I am not sure this was a correct change - I think I missed that there are two steps involved.